src/jit/hwintrinsicArm64.cpp

```diff
@@ -101,45 +94,45 @@ InstructionSet Compiler::lookupHWIntrinsicISA(const char* className)
 //
 // Return Value:
 //    Id for the hardware intrinsic.
-//
-// TODO-Throughput: replace sequential search by hash lookup
 NamedIntrinsic Compiler::lookupHWIntrinsic(const char* className, const char* methodName)
```

Comment:
This is basically identical to the x86 version now; but I wanted to just merge the `HWIntrinsicInfo` code in general in a follow up PR; rather than doing it piecemeal.